build/mingw32.cmake

```diff
@@ -4,8 +4,8 @@
 
 set (CMAKE_SYSTEM_NAME "Windows")
 
-set (CROSS /home/ws/mingw)
```

Comment:
You don't need to change this file. It's only used for crosscompiling the windows version on linux. Please revert the changes.